src/error.rs

```diff
@@ -20,6 +20,10 @@ pub enum Error {
     #[display(fmt = "Utf error: '{}'", _0)]
     Utf(std::str::Utf8Error),
 
+    #[cfg(feature = "tokio_support")]
+    #[display(fmt = "Join error: '{}'", _0)]
```

Comment:
We might use `#[display(transparent)]` here.

Reply:
Done.